static/js/app.js

```diff
@@ -847,6 +870,95 @@ $(function () {
             this.fetchMarkers();
             this.updateFilterString();
         },
+        loadFilterFromParameters: function() {
+            var bool_atrs = {};
+            bool_atrs["show_discussions"] = this.showDiscussion;
+            bool_atrs["show_markers"] = this.show_markers;
+            bool_atrs["accurate"] = this.accurate;
+            bool_atrs["approx"] = this.approx;
+            bool_atrs["show_fatal"] = this.show_fatal;
+            bool_atrs["show_severe"] = this.show_severe;
+            bool_atrs["show_light"] = this.show_light;
+
+            $.each(bool_atrs, function(attr, attr_value) {
+                if (attr_value == '1'){
```

Comment:
Sorry for being a little anal about it, but this is equivalent to simply doing

    $(attr).prop("checked", attr_value == '1');